test/uk/gov/hmrc/minorentityidentificationfrontend/services/StorageServiceSpec.scala

```diff
@@ -38,30 +38,30 @@ class StorageServiceSpec extends AnyWordSpec with Matchers with MockStorageConne
       "a sautr is entered" in {
         mockStorageConnector.retrieveOverseasDetails(testJourneyId) returns
           Future.successful(Some(OverseasCompanyDetails(Some(Sautr(testSautr)), Some(testOverseas))))
-        val result = await(TestStorageService.retrieveOverseasAuditDetails(testJourneyId))
+        val result = await(TestStorageService.retrieveOverseasAuditDetails(testJourneyId, testTrustJourneyConfig()))
```

Comment:
This probably won't impact the test but just for clarity sake could we use an Overseas Journey config here, I imagine there is one in the test constants file